GoRest.Api.Tests/Users/Get/GetAllUsers/GetAllUsersAfterAddingNewUserApiTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using API_Tests.Asserts;
+using GoRest.Api.Client.Client;
+using GoRest.Api.Client.Client.Builder;
+using GoRest.Api.Client.Client.Extentions;
+using GoRest.Api.Client.Client.Interfaces.Controllers;
+using NUnit.Framework;
+
+namespace GoRest.Api.Tests.Users
+{
+    [TestFixture]
+    public class GetAllUsersAfterAddingNewUserApiTests
+    {
+        [Test]
+        public async Task VerifyGetAllUsersReturnsIncreasedTotalAfterAddingNewUser()
+        {
+
+            // Arrange
+            var response = await GoRestClient.For<IUsersApi>().GetAll();
+            response.ShouldBeOK();
```

Comment:
make ShouldBeOk return value and use it at the same row as GetAll()

Reply:
Done

Reply:
Done